setup.py

```diff
@@ -21,8 +21,9 @@
         'paramiko==2.4.2',
         'pyyaml>=4.2b1',
         'jinja2==2.10.1',
-        'kubernetes',
+        'kubernetes<9.0.0,>=8.0.0',
```

Comment:
Can we add comment for the reasoning behind <9 >8

Reply:
Yes, I'll add a comment in. This was to satisfy the version needed for the `openshift` dependency.

Reply:
Fixed